hw02/EnemyCollision.cs

```diff
@@ -21,7 +21,14 @@ void Update()
     void Movement ()
     {
         // TODO: Problem 8: Please explain what the next 4 lines mean.
-        // What is Input.GetAxis, transform.Translate, and transform.Rotate? 
+        // What is Input.GetAxis, transform.Translate, and transform.Rotate?
+
+        //Input.GetAxis is the GetAxis() method within the old Input class from Unity.
+        //It takes in an axis as a parameter (normally defined in Unity's Project Settings).
+        //The axes have three values: -1 , 0, and 1
```

Comment:
Close! The GetAxis() will return -1 to 1. If you are on keyboard, it might just be 1 or -1, but if you are on controller move stick, it will be anything between -1...1.